CPP/Temperature_Conversions.cpp

```diff
@@ -0,0 +1,84 @@
+// Rahul Roy
+// Github Profile: https://github.com/rahulroy06/
+#include<iostream>
```

Comment:
Code is not working

Reply:
I just rechecked this code. and this is working completely fine.
If you are having any problem running this program. Then you can view it by link below and run it on that online compiler as well.

https://onlinegdb.com/ByiLjIy_w

Reply:
it's working